src/Helper/SupportRoomArchiver.js

```diff
@@ -25,15 +25,23 @@ const getRoomParticipants = async (supportRoom) => {
     return participants;
 }
 
+const determineLogChannel = function (message) {
+    const type = parseRoomType(message.channel.name);
+    let room = config.moderationLogsRoom;
+    if (typeof config.support.types[type].logRoom !== 'undefined') {
```

Comment:
Wouldn't `if (config.support.types[type].logRoom)` be the same?

Reply:
Yes.  Yes I think it would.